analysers/experiments/cpu.py

```diff
@@ -15,6 +15,16 @@
 from pyspark_cassandra import RowFormat
 from pyspark import SparkConf
 
+def getHostCores(sc, cassandraKeyspace, hostID):
```

Comment:
@VincenzoFerme I do, if the container properties doesn't have the number of cores I take it from the host. But keep in mind that Docker API returns a list of all cores on the machine regardless, with unused cores showing up with 0 usage. If I want to keep track of the indexes I need to save the whole array, and thus I need to know the number of cores on the host machine.